std_msgs/Header header

bool is_pos_valid
float64 latitude
float64 longitude

# Speed over ground, knots
float64 ground_speed

# Track made good, degrees True
float64 track_angle

# Date: dd/mm/yy
string date